manager/container.go

```diff
@@ -78,6 +83,17 @@ type containerData struct {
 	collectorManager collector.CollectorManager
 }
 
+// jitter returns a time.Duration between duration and duration + maxFactor * duration,
+// to allow clients to avoid converging on periodic behavior.  If maxFactor is 0.0, a
+// suggested default value will be chosen.
+func jitter(duration time.Duration, maxFactor float64) time.Duration {
```

Comment:
I've left the function signature, but defaulted on the call.

This should address the comments... I would like to leave the signature so I can rebuild and exercise during testing.  If at some point if it's deemed unnecessary we can easily collapse as it's a trivial change.